style.css

```diff
@@ -0,0 +1,274 @@
+:root {
+  --button-alpha-color: #25F18B;
+  --button-beta-color: #0ABCEB;
+}
+
+
+.wrapper--main-page {
+  background: radial-gradient(rgb(194, 74, 126), rgb(105, 86, 235));
+  padding-bottom: 240px;
+  padding-top: 40px;
+}
+
+.header {
+  align-items: center;
+  display: flex;
+  width: 1150px;
+}
+
+.header--navbar {
+  justify-content: space-between;
+  margin: 0 auto;
+  margin-bottom: 80px;
+}
+
+.header__logo {
+  padding-top: 20px;
+}
+
+.navbar__container {
+  color: white;
+  display: flex;
+}
+
+.navbar__item {
+  margin-right: 30px;
+}
+
+.section--main-page {
+  display: flex;
+  margin: 0 auto;
+  width: 1150px;
+}
+
+.section__container--plan {
```

Comment:
Nigdzie nie znalazłem deklaracji dla `.section__container`, a np. szerokość się powiela tj. w linii 101.
Powinno to wyglądać tak:
```
.section__container { /* cechy wspólne */
    width: 1150px;
    display: flex;
}

.section__container--plan { /* cechy szczególne */ 
   justify-content: center;
}

.section__container--features,
.section__container--team {
  flex-direction: row;
  margin: 0 auto;
}

```